### ISSUE TITLE
CPU usage

### ISSUE THREAD

**Issue by lorensr** [pranaygp/vscode-css-peek#38]
The below process was taking over 100% CPU continuously for a few days (and a new one appeared when I killed it) until I disabled this extension and restarted: 

```sh
$ ps aux | grep 38821
me               38821 191.6 13.3  7302104 2232572   ??  R     2:01AM   2:07.45 /Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper.app/Contents/MacOS/Code Helper /Users/me/.vscode/extensions/pranaygp.vscode-css-peek-2.1.0/node_modules/vscode-languageclient/lib/utils/electronForkStart /Users/me/.vscode/extensions/pranaygp.vscode-css-peek-2.1.0/server/server.js --node-ipc --clientProcessId=1425
me               38900   0.0  0.0  4276984    916 s004  S+    2:03AM   0:00.00 grep 38821
```

**Comment by suhao** [comment]
Hi,
I have the same problem.
I used VSCode(1.29.1, system setup) on windows10 + CSS Peek 2.2.0.
The CSS Peek settings is the default ones.
The system's CPU will be nearly 100% and I always need to restart the VSCode.
Our project include the source code of [boost](https://www.boost.org/users/history/version_1_68_0.html), which had many html and xml files.

I found that enabled this extension, will open many instance of rg.exe. Will this extension could keep the number of the rg.exe instance?

**Comment by marcinkrzeminski** [comment]
Having the same issue for a couple of days. Had to disable the extension ;(

**Comment by killerlei** [comment]
I also have the same problem;
my envirnment is mac os 10.13.6 + vscode 1.30.2,
intersting! 
 IntelliSense for CSS class names in HTML----other one vscode extension on css have the same problem

**Comment by pranaygp** [comment]
This is resolved in CSS Peek 4.0.0

**Comment by mitivil** [comment]
**vs.code css-peek-4.2.0**
, it still heavily overloads the processor and memory, attached is a screenshot from Linux-mint manager.
This process (vs.code css code-peak-4.2.0) does not terminate in the usual way of closing the application, you have to kill manually and it hangs in the background ((( sad.

**P/s (css-peek) I assume that this is a viral add-on for vs.code, it is either engaged in mining, or simply not optimized.**
